test/e2e/packetcapture_test.go

```diff
@@ -270,9 +271,9 @@ func testPacketCaptureBasic(t *testing.T, data *TestData) {
 				Conditions: []crdv1alpha1.PacketCaptureCondition{
 					{
 						Type:               crdv1alpha1.PacketCaptureComplete,
```

Comment:
so the capture can be "complete" even though it never "started"?
I think this is related to what you brought up earlier: the only way to know if the capture was successful is to look at the `Reason` field for the `Complete` condition. It can be one of `Failed`, `TImeout`, `All packets captured`?

Reply:
Right, there may be some space to improve, for now I just use how far the request has been processed to determine the condition. If it ever "startCapture", it's "started", and it will be "complete" once "startCapture" returns.